hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -239,4 +240,31 @@ public void run()
             }
         });
     }
+
+    @Test
+    public void testListNodes() throws IOException {
+        final HazelcastInstance instance1 = Hazelcast.newHazelcastInstance(config);
+        HTTPCommunicator communicator = new HTTPCommunicator(instance1);
+
+        assertTrue(instance1.getLifecycleService().isRunning());
+        assertEquals("{\"status\":\"success\" \"response\":\"[" + instance1.getCluster().getLocalMember().toString() + "]\n" +
```

Comment:
To make this a bit easier to read I suggest to use a String.format. Java is very noisy with string concatenation and makes it hard to read the output.